### DISCUSSION TITLE
Editable homepage

### DISCUSSION THREAD

**Discussion by tomjowitt** [outline/outline#2223] [Ideas]
Hey there,
Really enjoying using Outline so far. It's a breath of fresh air. Great work team.

One thing that would really improve our experience would be having the homepage as an editable page like any other so we could build it out as a central hub for all of our work.

Is this something that you would consider?

**Comment by BurgosNY** [comment]
I'd love to see that as well. The product is amazing, but I think the homepage, with a feed-like experience, might be confusing for some.

**Comment by lampy255** [comment]
I too would love to see this as a feature.
Similar or the same as discussion #4892

**Comment by ThinkSalat** [comment]
Agreed, in fact this would perfectly fit my own system of PKM. I keep a central hub and branch my projects/ideas and other such items into their own pages once they begin to take too much space in the main page.
